### ISSUE TITLE
H5Z-ZFP binding?

### ISSUE THREAD

**Issue by stillyslalom** [milankl/ZfpCompression.jl#21]
It seems like this would be the logical repository for a binding to the [H5Z-ZFP](https://github.com/LLNL/H5Z-ZFP) HDF5 filter plugin, perhaps as a separate package in a subdirectory to match the [filters](https://github.com/JuliaIO/HDF5.jl/tree/master/filters) in the main HDF5.jl repo.

**Comment by milankl** [comment]
I agree that sounds like a good idea. I haven't worked much with HDF5, but I'm happy to review and merge pull requests if someone wants to including the bindings here!
